### ISSUE TITLE
SC1009 Parser Error

### ISSUE THREAD

**Issue by DonRichards** [koalaman/shellcheck#746]
## How do I fix or bypass this error without doing it globally?

* I'm running OSX (Shellcheck passes) commands line & IDE plugin
* When it runs on Travis-CI / Ubuntu (Shellcheck fails)

custom.sh

```bash
#!/usr/bin/env bash
# run all shell scripts in scripts/custom_scripts/

# shellcheck source=/dev/null
for SCRIPT in /vagrant/scripts/custom_scripts/*.sh
do
 if [ -f "$SCRIPT" ]; then
  echo "Running custom scripts"
  # shellcheck source=/dev/null
  source "$SCRIPT"
 fi
done
```

## Results

```shell
$ bash -c 'shopt -s globstar; shellcheck **/*.{sh,bash}'
In scripts/custom.sh line 4:
for SCRIPT in /vagrant/scripts/custom_scripts/*.sh
^-- SC1009: The mentioned parser error was in this for loop.
In scripts/custom.sh line 6:
 # shellcheck source=/dev/null
 ^-- SC1073: Couldn't parse this shellcheck annotation.
              ^-- SC1072: Unexpected "s". Fix any mentioned problems and try again.
**/*.bash: openFile: does not exist (No such file or directory)
The command "bash -c 'shopt -s globstar; shellcheck **/*.{sh,bash}'" exited with 2.
```

**Comment by koalaman** [comment]
Which versions are you using on the two platforms?

**Comment by DonRichards** [comment]
OSX: 
Terminal: 
ShellCheck - shell script analysis tool
version: 0.4.4
license: GNU General Public License, version 3
website: http://www.shellcheck.net

Atom 1.11.1
Linter-shellcheck 1.4.1

Trasvis-CI: 
ShellCheck - shell script analysis tool
version: 0.3.3
license: GNU Affero General Public License, version 3
website: http://www.shellcheck.net

**Comment by koalaman** [comment]
That would explain it. Following sourced files was not added until after 0.3.3. Making failing directives a parser error was not a good idea.

**Comment by koalaman** [comment]
In 88c56ec this will no longer be a parser error, just a warning. For now, please see if you can update from 0.3.3 to something newer on Travis-CI.
